ozaria/site/components/teacher-dashboard/BaseMyClasses/ClassComponent.vue

```diff
@@ -24,17 +25,49 @@ export default {
   },
 
   computed: {
+    ...mapGetters({
+      tournamentsByClan: 'clans/tournamentsByClan',
+      tournaments: 'clans/tournaments',
+      allTournamentsLoaded: 'clans/allTournamentsLoaded'
+    }),
+
     showEsportsCampInfoCoCo () {
       return utils.isCodeCombat && me.isCodeNinja() && !this.chapterStats.length
     },
 
+    me () {
+      return me
+    },
+
```

Comment:
Is this how we do this, vs. using a vuex store `me`? Seems ok but wonder if this would make `me` reactive when it wasn't before. (maybe this is ok)

Reply:
this just add window.me to vue context. i think it would be okay. vuex me should be okay too?